### PR TITLE
removing pesky period

### LINKED ISSUE #247: Multiple books => Extra period in appendix table numbers
EDIT: this actually affects more books than initially thought, but originally spotted in AP Physics.

Inside the AP Physics appendices, the table numbers have an extra period that should be removed, i.e. the table number should be `Table A1`, not `Table A.1`, in order to match the PDF. 

https://dev.cnx.org/contents/jQSmhtXo@14.5:cjQyhWg-@7/Appendix-C-Useful-Information

image

### PR DESCRIPTION
**Pull request by caderitter**
to fix #247

### CODE REVIEW

**Comment by kerwinso** [comment]
This was originally reported for one book in #247, but looks like it actually affects multiple books.

Checklist for QA:

- [ ] TEA AP Bio
- [ ] TEA AP Physics
- [ ] TEA Econ
- [ ] TEA HS Physics
- [ ] TEA Statistics
- [ ] AP Bio
- [ ] AP Physics
- [ ] College Biology 
- [ ] Econ
- [ ] College Physics
- [ ] Statistics

**Comment by philschatz** [comment]
Sorry about that Review Request @kerwinso . I'm still getting used to clicking "Assignees" 😊

File: rulesets/books/_common-stuff.scss [hunk 1]
```diff
@@ -35,8 +35,8 @@ $Config_TargetLabels: (
   (selector: '[data-type="chapter"] :not(figure) > figure:not(.unnumbered)',  labelText: "Figure " counter(chapter) "." counter(figure)),
   (selector: '.appendix :not(table) > table:not(.unnumbered)',   labelText: "Table " counter(appendix, upper-alpha) "." counter(table)),
   (selector: '.appendix :not(figure) > figure:not(.unnumbered)', labelText: "Figure " counter(appendix, upper-alpha) "." counter(figure)),
-  (selector: '.appendix > table:not(.unnumbered)', labelText: "Table " counter(appendix, upper-alpha) "." counter(table)),
-  (selector: '.appendix > figure:not(.unnumbered)', labelText: "Figure " counter(appendix, upper-alpha) "." counter(figure)),
+  (selector: '.appendix > table:not(.unnumbered)', labelText: "Table " counter(appendix, upper-alpha) "" counter(table)),
+  (selector: '.appendix > figure:not(.unnumbered)', labelText: "Figure " counter(appendix, upper-alpha) "" counter(figure)),
```
Comment (tomjw64) [inline rulesets/books/_common-stuff.scss]: Do we still need those empty quotes `""`?

**Comment by caderitter** [comment]
@kerwinso #247

**Comment by kerwinso** [comment]
Cade says this addresses #251 as well.

**Pull request merged**